### ISSUE TITLE
Internationalisation

### ISSUE THREAD

**Issue by RaphaelPage0110** [orichalque/yet-an-other-gloomhaven-companion#92]
It could be nice to translate the app (in French at first)...If we find the right images!

**Comment by brettmcginnis** [comment]
If you wanted to do this it's fairly straight forward. Centralize all strings used, into a json. And then make another json object for each supported language. 

https://www.i18next.com/

**Comment by orichalque** [comment]
Yes, the hard part is finding the card pictures in the right language ^^
